packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -3678,6 +3678,40 @@ Content-Type: application/json
 }
 ```
 
+### Update application policy
+
+<ApiOperation method="put" url="/api/v1/apps/${applicationId}/policies/${policyId}" />
```

Comment:
This is a `PUT` rather than a `POST`? Little unusual to have a `PUT` without a body so just wanted to confirm

Reply:
it is put , and the call is working without body, I can look into that more

Reply:
![image](https://user-images.githubusercontent.com/84338599/155601301-9006afa4-99b6-4673-b895-8eeddf41fe62.png)